src/pages/index.astro

```diff
@@ -2,241 +2,177 @@
 import data from "../data/home.json";
-import contributors from "../data/contributors.json";
 import CardList from "../components/pages/CardList.astro";
-import PrCta from "../components/pages/PrCta.astro";
 
 const lang = getLangFromUrl(Astro.url);
 const t = useTranslations(lang);
-const pluginsList = data.plugins.filter(function(plugin){
-  return plugin.version.includes("2.0");
-});
 ---
 <Base title={t('site.title')} description={t('site.description')}>
   <section class="section hero">
-    <div class="contentWrapperSm">
-      <img
-        alt="Web Monetization Icon"
-        class="heroIcon"
-        src="/img/wm-logo--animated.svg"
-        width="120"
-        height="86"
-      />
-      <h1 class="heading2">Web Monetization</h1>
-      <p class="heroTagline heading4">
-        The Web Monetization API allows websites to automatically and
-        passively receive payments from Web Monetization-enabled visitors.
-      </p>
-    </div>
-    <footer class="heroFooter">
-      <div class="contentWrapperSm">
-        <p>
-          Web Monetization is being proposed as a standard in <a
-            href="https://wicg.io/"
-            data-umami-event="Landing page - Spec"
-          >
-            the W3C Web Incubator Community Group</a>.
-        </p>
-      </div>
-    </footer>
-  </section>
-  <section class="section">
-    <div class="contentWrapperLg twoColGrid">
-      <div class="gridItem">
-        <h2 class="heading3">Motivation</h2>
-        <p>
-          The ability to transfer money has been a long-standing omission
-          from the web platform. As a result, the web suffers from a flood
-          of advertising and broken business models. Web Monetization
-          provides an open, native, efficient, and automatic way to
-          compensate creators, pay for content, and support crucial web
-          infrastructure.
-        </p>
-      </div>
-      <div class="gridItem">
-        <h2 class="heading3">Why Now?</h2>
-        <p>
-          Until recently, there hasn't been an open, neutral, and cost-efficient protocol for transferring money. <a href="https://interledger.org/" data-umami-event="Landing page - ILP docs">Interledger</a> <a href="https://openpayments.dev/" data-umami-event="Landing page - Open Payments docs">Open Payments</a> provides a simple, interoperable, and currency-independent method for the transfer of small amounts of money. This opens up the possibility for streaming money, which makes the Web Monetization use case possible for the first time.
-        </p>
+    <div class="contentWrapper">
+      <div class="heading-wrapper">
+        <h1>
+          Welcome to Web Monetization
+        </h1>
+        <img
+          alt=""
+          role="presentation"
+          src="/img/wm-heading-vector.svg"
+          width="442"
+          height="23"
+        />
       </div>
+      <video
+        class="hero-video"
+        src="/video/hero-animation.webm"
```

Comment:
Consider adding a `poster` attribute to show an image until video loads, making page look more complete, and potentially avoid a large LCP.